source-hubspot-native/source_hubspot_native/api.py

```diff
@@ -122,6 +122,121 @@ async def fetch_page(
         yield result.paging.next.after
 
 
+async def fetch_page_incremental(
+    cls: type[CRMObject],
+    http: HTTPSession,
+    log: Logger,
+    log_cursor: LogCursor,
```

Comment:
Can we use this `log_cursor` as the initial input value for `after`? The idea being that otherwise every time this function is called, it will restart from the "beginning" of the API. But we will already know that we have read up to the time of the `log_cursor` so we don't need to get anything older than that.

Reply:
The V3 endpoint only allows for page tokens to be inserted on `after`. The stream would always re-start and bigger users would be affected without a actual "full_refresh".
The stream backoff is really fast, setting it up as default in the ResourceState should take care of real time data for bigger users @williamhbaker ?

Reply:
Ah ok so after is an integer-ish page token?

Reply:
@williamhbaker yes, its a random string